test/functional/commands/jetpack-compose-e2e-specs.js

```diff
@@ -0,0 +1,39 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { initSession, deleteSession, MOCHA_TIMEOUT } from '../helpers/session';
+import { COMPOSE_CAPS } from '../desired';
+
+
+chai.should();
+chai.use(chaiAsPromised);
+
+describe('Jetpack Compose', function () {
+  this.timeout(MOCHA_TIMEOUT);
+
+  let driver;
+  before(async function () {
+    driver = await initSession(COMPOSE_CAPS);
+  });
+  after(async function () {
+    await deleteSession();
+  });
+
+  it('should find element by tag and text and click it', async function () {
+    let el = await driver.elementByXPath("//*[@text='Clickable Component']");
+    await el.click();
+
+    await driver.updateSettings({ driver: 'compose' });
```

Comment:
please also add a validation that verifies changed settings retrieval

Reply:
It was already there as the last verification